frontend/integration-tests/views/olm-catalog.view.ts

```diff
@@ -0,0 +1,27 @@
+import { browser, $, $$, by, element, ExpectedConditions as until } from 'protractor';
+
+import * as crudView from './crud.view';
+
+const rowSelector = '.co-resource-list__item';
+export const entryRows = $$(rowSelector);
+export const entryRowFor = (name: string) => element(by.cssContainingText(rowSelector, name));
```

Comment:
This is going to have to be reworked when #1465 merges.

Reply:
Updated.